src/main/java/edu/project/components/GeoMap.java

```diff
@@ -5,40 +5,43 @@
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
 import java.awt.event.MouseListener;
+import java.io.IOException;
 import java.util.HashSet;
-
 import javax.swing.border.EmptyBorder;
-
 import org.jxmapviewer.JXMapKit;
 import org.jxmapviewer.JXMapViewer;
 import org.jxmapviewer.input.CenterMapListener;
 import org.jxmapviewer.viewer.DefaultWaypoint;
 import org.jxmapviewer.viewer.GeoPosition;
 import org.jxmapviewer.viewer.WaypointPainter;
-
 import edu.project.Config;
 import edu.project.Context;
 import edu.project.tilesets.HybridTileFactory;
+import edu.project.yourpackage.WeatherCurrentData;
+import edu.project.yourpackage.WeatherForecast5Data;
 
 public class GeoMap extends JXMapKit {
+
+	private Context context;
+	private static final int MAP_INIT_ZOOM = 10;
```

Comment:
No need to store `context` since its only used within the constructor, and the `MAP_INIT_ZOOM` constant can be accessed through the `Config` class